chain-db-open-server/CHANGELOG.md

```diff
@@ -0,0 +1,138 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## [0.5.1](https://github.com/paritytech/Nomidot/compare/v0.5.0...v0.5.1) (2020-05-12)
```

Comment:
needs to be changed to https://github.com/paritytech/polkassembly

Reply:
I'll remove the changelog alltogether